test/terraform_jx_cluster.go

```diff
@@ -0,0 +1,69 @@
+package test
```

Comment:
Shouldn't this file be named like `*_test.go`?

Reply:
Yep - mistake on my part :)